src/type_conversion/types.h

```diff
@@ -0,0 +1,49 @@
+#ifndef MY_TYPES_H
+#define MY_TYPES_H
```

Comment:
Any reason not to use `stdint.h` instead?

Reply:
Because `uint32_t` is kind of a mouthful, especially the underscore. Since Rust and Linux both use `u32` I feel like there is a precedent. Maybe I should include stdint and then `typedef uint32_t u32;`

Reply:
`typedef uint32_t u32` is definitely an improvement imo. I think strictly speaking, the C standard doesn't specify what e.g. `int` must be, so `uint32_t` would be more explicit and more obviously correct.

wasm uses these type names too so I would definitely consider them standard

Reply:
Great, I'll do that.

Reply:
On the other hand, there is also advantage in using more standard definitions in terms consistency, especially since these types are not used a lot. I don't have strong opinions either way.

Reply:
> strictly speaking, the C standard doesn't specify what e.g. `int` must be

Absolutely, this is an annoying deficiency of the C language (it's even worse in OpenGL though).